core/src/components/checkbox/checkbox.ionic.vars.scss

```diff
@@ -0,0 +1,5 @@
+@import "../../themes/ionic.globals.ios";
+@import "../item/item.ios.vars";
```

Comment:
Why are we importing the iOS styles here?

Reply:
Good question, sloppy copy & paste on my behalf with splitting up the original PR. 

I posted the question in an internal DM, but I don't believe we should be referencing any iOS/MD variables (global or component specific) in the Ionic theme. The ionic theme should be completely decoupled from the other themes. 

The value of this PR is we caught that, because the scope is small 😉